src/pages/XmlUpload/components/DropZone/DropZone.jsx

```diff
@@ -20,27 +20,71 @@ const DropZone = ({ files, setFiles }) => {
     });
   };
 
+  const CONVERSION = 1e6; // = 1,000,000
+
+  const escapeRegex = (string) => string.replace(/[-/\\^$*+?.()|[\]{}]/g, '\\$&');
+
+  const fixDuplicateFile = (updatedFiles, file) => {
```

Comment:
```suggestion
  const renameDuplicateFile = (updatedFiles, file) => {
```

This is a better name